src/utils/breakpoints.js

```diff
@@ -1,27 +1,32 @@
 // NOTE: These are the breakpoints used in Bootstrap v4.0.0 as seen in
 // the documentation (https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints)
+
+import {
+  sm, md, lg, xl, xxl,
+} from '../../scss/core/_extend.module.scss';
+
 const breakpoints = {
   extraSmall: {
-    maxWidth: 575.98,
+    maxWidth: parseFloat(sm) || 575.98,
```

Comment:
Hardcoded values are used to prevent tests to not fail. In the tests there is no possibility to read `_extend.module.scss` that results that `parseFloat(sm)` results in `NaN`. Test are based on the breakpoints fails then. If we add `console.error` probably we will receive all these errors in the tests. So I don't know if it is okay for us

Reply:
Interesting. Any leads on why it's not possible to read `_extend.module.scss`? This might be worth exploring more. Regardless, I'm OK to merge with the fallbacks for the time being.